app/controllers/administration_controller.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class AdministrationController < ApplicationController
+  # binding.pry
```

Comment:
Useless comment here.

Reply:
yes & no, pry is Life. will be removed after